File: 476_Number Complement.swift
class Solution {
    func findComplement(_ num: Int) -> Int {
        var n = num
        var k = ""
        while n != 0 {
            k = k + String(n % 2)
            n = n/2
        }
        k = String(k.characters.reversed())
        var result = ""
        for c in k.characters {
            if c == "1" {
                result += "0"
            }else{
                result += "1"
            }
        }
        return Int(result, radix: 2)!
    }
}

File: 476_Number Complement.java
public class Solution {
    public int findComplement(int num) {
        String bstring = Integer.toBinaryString(num);
        char[] bs=bstring.toCharArray();
        String bsbs="";
        for(int i=0;i<bs.length;i++)
        {
            if(bs[i]==48)bsbs=bsbs+"1";
            else
            bsbs=bsbs+"0";
        }
        
        return Integer.valueOf(bsbs,2);
    }
}
